frontend/src/app/spot/components/selector-field/stories/SelectorField.example.html

```diff
@@ -0,0 +1,46 @@
+<div class="spot-container">
+  <spot-selector-field [label]="'not checked'">
```

Comment:
```suggestion
  <spot-selector-field label="not checked">
```

All of these can be simplified because you're inputting strings

Reply:
fixed